wordpress-mobile/WordPress-iOS
# Adds IA Navigation announcement modal

#13788 

Adds a modal to announce new IA changes. I have a couple of questions:

* Do we need to track that this modal was shown? Do we need to track when the "Read More" button is tapped?
* Do we need more advanced logic to hide the modal after a certain amount of time has passed after the release so that we aren't showing users this prompt 2 years from now? Alternatively, we could remove it in a future release.

<a href="https://user-images.githubusercontent.com/3250/80828524-a789a500-8ba2-11ea-9fa6-681883e60980.png"><img src="https://user-images.githubusercontent.com/3250/80828524-a789a500-8ba2-11ea-9fa6-681883e60980.png" width="300"></a>


To test:
- Open the app and notice the IA Navigation announcement modal
- Tap on the "Read more" button to navigate to the wordpress blog (blog URL to be added separately once post is created)


#### Resetting the modal state

To reset the User Defaults key so that the modal appears again pause using the debugger and run the following command:

```
e -l swift -- import WordPress; UserDefaults.standard.createButtonAlertWasDisplayed = false
```

After doing this, the modal should appear on next launch.

PR submission checklist:
